### PR TITLE
TELCODOCS-1036: Incorporated additional SME feedback.

### PR DESCRIPTION
**Pull request by johnwilkins**
Incorporated additional SME feedback. 

Version(s): 4.13

Issue: https://issues.redhat.com/browse/TELCODOCS-1036

Link to docs preview: http://jowilkin.com:8080/TELCODOCS-1036/installing/installing_aws/installing-aws-expanding-a-cluster-with-on-premise-bare-metal-nodes.html

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by tomassedovic** [comment]
Thank you @johnwilkins! LGTM

**Comment by elfosardo** [comment]
/lgtm
thanks @johnwilkins

**Comment by johnwilkins** [comment]
@tmalove Any changes?

File: installing/installing_aws/installing-aws-expanding-a-cluster-with-on-premise-bare-metal-nodes.adoc [hunk 1]
```diff
@@ -6,9 +6,9 @@ include::_attributes/common-attributes.adoc[]
...
+Expanding an {product-title} cluster deployed on AWS requires using virtual media with bare metal nodes that meet the xref:../installing_bare_metal_ipi/ipi-install-prerequisites.adoc#node-requirements_ipi-install-prerequisites[node requirements] and xref:../installing_bare_metal_ipi/ipi-install-prerequisites.adoc#ipi-install-firmware-requirements-for-installing-with-virtual-media_ipi-install-prerequisites[firmware requirements] for installing with virtual media. A `provisioning` network is not required, and if present, should be xref:../installing_bare_metal_ipi/ipi-install-installation-workflow.adoc#modifying-install-config-for-no-provisioning-network_ipi-install-installation-workflow[disabled].
```
Comment (tmalove) [inline installing/installing_aws/installing-aws-expanding-a-cluster-with-on-premise-bare-metal-nodes.adoc:11]:
```suggestion
Expanding an {product-title} cluster deployed on AWS requires using virtual media with bare-metal nodes that meet the xref:../installing_bare_metal_ipi/ipi-install-prerequisites.adoc#node-requirements_ipi-install-prerequisites[node requirements] and xref:../installing_bare_metal_ipi/ipi-install-prerequisites.adoc#ipi-install-firmware-requirements-for-installing-with-virtual-media_ipi-install-prerequisites[firmware requirements] for installing with virtual media. A `provisioning` network is not required, and if present, should be xref:../installing_bare_metal_ipi/ipi-install-installation-workflow.adoc#modifying-install-config-for-no-provisioning-network_ipi-install-installation-workflow[disabled].
```

File: modules/installation-aws_con_connecting-the-vpc-to-the-on-premise-network.adoc [hunk 2]
```diff
@@ -14,5 +14,5 @@ To securely access the BMCs, you can create a separate, secure network segment o
 
 [WARNING]
 ====
-Misconfiguration of the network connection between the AWS and on-premise environments can expose the on-premise network and bare metal nodes to the internet.
+Misconfiguration of the network connection between the AWS and on-premise environments can expose the on-premise network and bare metal nodes to the internet. That is a significant security risk and could result in an attacker having full access to the exposed machines, and through them the private network in these environments.
```
Comment (tmalove) [inline modules/installation-aws_con_connecting-the-vpc-to-the-on-premise-network.adoc:17]:
```suggestion
Misconfiguration of the network connection between the AWS and on-premise environments can expose the on-premise network and bare-metal nodes to the internet. That is a significant security risk, which might result in an attacker having full access to the exposed machines and through them to the private network in these environments.
```
See the ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#c

**Review by tmalove — APPROVED** [review]

**Pull request merged**